scripts/ci/docker-compose/integration-mssql.yml

```diff
@@ -35,7 +35,7 @@ services:
       test: /opt/mssql-tools/bin/sqlcmd -S localhost,1433 -U sa -P "airflow123" -Q "SELECT 1" -b -o /dev/null
       interval: 10s
       timeout: 3s
-      retries: 10
+      retries: 30
```

Comment:
Because I saw some of the tests failing - see https://github.com/apache/airflow/pull/39877 - but it still did not help. It's a bit of a guess what values will be enough for the Public runners.